packages/genesys-spark-components/src/components/beta/gux-time-zone-picker/tests/gux-time-zone-picker.spec.ts

```diff
@@ -10,6 +10,8 @@ describe('#render ', () => {
     `<gux-time-zone-picker-beta value="Etc/GMT+1" custom-default="America/Detroit" custom-default-label="Business Timezone"><gux-time-zone-picker-beta>`
   ].forEach((html, index) => {
     it(`should render component as expected (${index + 1})`, async () => {
+      jest.useFakeTimers({ advanceTimers: true });
+      jest.setSystemTime(new Date('2023-09-26'));
```

Comment:
This date was already used in a different timezone related test file so I used it here too.